front/src/routes/history/style.css

```
@@ -1,3 +1,55 @@
+   ================================================================== */
+
+/* Sticky metrics shared by the filter capsule, the day labels and the
+   "new events" pill. Below the desktop breakpoint the fixed mobile top
+   bar (3.25rem, components/header) is cleared; on coarse pointers the
+   chips grow to the ~44px touch floor like the dashboard selector pills. */
+.historyPage {
+  --hz-chip-h: 2.25rem;
+  /* capsule outer height: chip + 2 × 5px padding + 2 × 1px border */
+  --hz-capsule-h: calc(var(--hz-chip-h) + 12px);
+  --hz-sticky-top: 0.75rem;
+  min-height: 100vh;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)